apps/email-slack-app/src/lambda.ts

```diff
@@ -0,0 +1,44 @@
+import * as Sentry from "@sentry/aws-serverless"
+import { nodeProfilingIntegration } from "@sentry/profiling-node"
+import type { APIGatewayProxyEventV2, APIGatewayProxyResultV2, Handler } from "aws-lambda"
+import { createExternalClients, createServiceLayer } from "./core"
+import { env } from "./env"
+
+Sentry.init({
+  dsn: "https://a07a847a3bbab26aac0ea417edcbc7f2@o4504113989287936.ingest.us.sentry.io/4508080738467840",
```

Comment:
Use an environment variable for this, and disable it in process.env.NODE_ENV !== production

Reply:
Yeah good idea, thanks